Duplicati/Server/WebServerLoader.cs

```diff
@@ -57,6 +64,16 @@ public static class WebServerLoader
     /// </summary>
     public const int DEFAULT_OPTION_PORT = 8200;
 
+    /// <summary>
+    /// The default certificate file for https
+    /// </summary>
+    public const string DEFAULT_OPTION_CERTIFICATEFILE = "server-webui.pfx";
+
+    /// <summary>
+    /// Option for setting if to use HTTPS
+    /// </summary>
+    public const string OPTION_USEHTTPS = "webservice-usehttps";
```

Comment:
This is slightly confusing. Is this meant to support disabling HTTPS without deleting the certificate? If so, perhaps we can do something else. Maybe just have it inverted to `webservice-disablehttps`? 

Or is it meant to be a safguard the prevents the server to start without a valid SSL configuration?

I think there are a few places that check for the certificate and they all need to be updated with this logic.

Reply:
Yes it was meant to be support for disable HTTPS without deleting certificate.